src/pivotal-ui/components/header/header.scss

```diff
@@ -192,8 +192,23 @@ header {
     background-color: $header-secondary;
   }
 
+  .drop-down-menu-toggle::after {
+    display: none;
+  }
+  .drop-down-menu-toggle.toggle-is-open svg {
+    transform: rotate(180deg);
+  }
+
 }
 
+.header-drop-down-icon {
+  // Like the .icon class, but duplicated here because this has to work on non-component pages.
```

Comment:
We could! Or, we could in fact include the icon class as a dependency, and import it in the website, so we hand-pluck out two things, not just one.